lib/docx/document.rb

```diff
@@ -38,7 +38,7 @@ def initialize(path_or_io, options = {})
       load_styles
       yield(self) if block_given?
     ensure
-      @zip.close
+      @zip.close unless @zip.blank?
```

Comment:
`blank?` is provided by `rails`  so should be `nil?` here

```suggestion
      @zip.close unless @zip.nil?
```